use_cases/CFD_example.gid/test_kratosWrapper.py

```diff
@@ -39,7 +49,7 @@
 CFDWrapper.setMeshData(mesh)
 CFDWrapper.add_mesh(mesh)
 
-for i in xrange(0,10):
+for i in xrange(0,CFDWrapper.CM[CUBA.NUMBER_OF_TIME_STEPS]):
```

Comment:
I think the '''CFDCFDWrapper.CM[CUBA.NUMBER_OF_TIME_STEPS]''' should
be describing the number of time steps that occur when '''run''' occurs.  

For example:

```
CFDWrapper.CM[CUBA.NUMBER_OF_TIME_STEPS] = 5
for i in xrange(0, 10):
  CFDWrapper.run()  # runs 5 steps

#has now run a total of 50 steps
```